plugins/toolkit/jetbrains-core/tst/software/aws/toolkits/jetbrains/services/codewhisperer/codescan/CodeWhispererCodeFileScanTest.kt

```diff
@@ -168,6 +169,15 @@ class CodeWhispererCodeFileScanTest : CodeWhispererCodeScanTestBase(PythonCodeIn
         )
     }
 
+    @Test
```

Comment:
Why is the same test written twice?

Reply:
I thought to write test for both File scan and project scan but removed this.